README.md

```diff
@@ -40,6 +40,35 @@ val client = S3AsyncClient
               .region(Region.EU_CENTRAL_1)
               .httpClient(akkaClient)
               .build()
+
+val eventualResponse = client.listBuckets()
+```
+
+If you need to connect to some AWS service from internal Corporate, where is needed to configure a Proxy, it can be performed in the following way:
```

Comment:
```suggestion
If you connect to an AWS service from inside a corporate network, it may be necessary to configure a proxy. This can be achieved in the following way:
```